deploy/toolchaincluster/member-sa.yaml

```
@@ -0,0 +1,109 @@
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: toolchaincluster-member
+  namespace: {{.Namespace}}
+rules:
+- apiGroups:
+  - toolchain.dev.openshift.com
+  resources:
+  - "idlers"
+  - "nstemplatesets"
+  - "memberoperatorconfigs"
+  - "memberstatuses"
+  - "toolchainclusters"
+  - "useraccounts"
+  verbs:
+  - "*"
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: toolchaincluster-member-{{.Namespace}}-toolchaincluster
+rules:
+- apiGroups:
+  - authentication.k8s.io
+  resources:
+  - tokenreviews
+  verbs:
+  - create
+- apiGroups: [""]
+  resources: ["users", "groups"]
+  verbs: ["impersonate"]
```

Or maybe just for consistency's sake so we use the same format in this template:
```suggestion
- apiGroups:
  - ""
  resources:
  - users
  - groups
  verbs:
  - impersonate
```

---

thanks!